erroranalysis/erroranalysis/_internal/error_analyzer.py

```diff
@@ -82,6 +83,20 @@ def compute_error_tree(self, features, filters, composite_filters):
         return compute_json_error_tree(self, features, filters,
                                        composite_filters)
 
+    def compute_importances(self):
```

Comment:
Always good to have docstrings for public methods

Reply:
yes will add doc strings later, right now this is all in _internal so it's all private anyway